phoenix-core/src/main/java/org/apache/phoenix/execute/AggregatePlan.java

```diff
@@ -327,4 +333,15 @@ public boolean useRoundRobinIterator() throws SQLException {
         return visitor.visit(this);
     }
 
+    private static OrderBy convertActualOutputOrderBy(OrderBy orderBy, GroupBy groupBy, StatementContext statementContext) {
```

Comment:
Does every plan have to implement this function, with different logic?

Reply:
Not every plan have to  implement this function, eg. HashJoinPlan and SortMergeJoinPlan.
In my opinion, this method is not suitable to declared as abstract method in QueryPlan and subclass to override it , because this method is to invoke in the ctor of concrete  subclass.